packages/mdc-icon-button/_mixins.scss

```diff
@@ -79,7 +81,28 @@
   }
 }
 
-@mixin mdc-icon-button-size($width, $height: $width, $padding: max($width, $height)/2, $query: mdc-feature-all()) {
+@mixin mdc-icon-button-size($size, $query: mdc-feature-all()) {
+  $feat-structure: mdc-feature-create-target($query, structure);
+
+  @include mdc-feature-targets($feat-structure) {
+    width: $size;
+    height: $size;
+    padding: $size / 4;
+    font-size: $size / 2;
```

Comment:
We do not change the size of font icon or image icon when density is applied. So this'll remain at original size (ie., 24px).

The minimum density size that is supported would be 32px (Density scale -4).

Reply:
Updated!